### PR TITLE
5192 fixes invalid cloudformation

### PR DESCRIPTION
**Pull request by robertfreund**
Fixes https://github.com/kubernetes/kops/issues/5192

### CODE REVIEW

**Comment by idealhack** [comment]
@robertfreund Thanks for your contribution! Please run `gofmt` and amend this commit, also, please sign the CLA.

File: upup/pkg/fi/cloudup/awstasks/elastic_ip.go [hunk 1]
```diff
@@ -304,7 +304,6 @@ type cloudformationElasticIP struct {
 func (_ *ElasticIP) RenderCloudformation(t *cloudformation.CloudformationTarget, a, e, changes *ElasticIP) error {
 	tf := &cloudformationElasticIP{
 		Domain: aws.String("vpc"),
-		Tags:   buildCloudformationTags(e.Tags),
```
Comment (justinsb) [inline upup/pkg/fi/cloudup/awstasks/elastic_ip.go]: Hmm that's disappointing  - does cloudformation not support tagging ElasticIPs?  We should probably warn if we're dropping tags here.  Or can we do this with a different cloudformation task?

Comment (robertfreund) [reply upup/pkg/fi/cloudup/awstasks/elastic_ip.go]: this still fails. Can we get this accepted?

This is output when run: Encountered unsupported property Tags

**Pull request closed without merging**